Soundex.Tests.cpp

```diff
@@ -1,9 +1,68 @@
 #include <gtest/gtest.h>
 #include "Soundex.h"
 
-TEST(SoudexTestsuite, ReplacesConsonantsWithAppropriateDigits) {
- //AAA
-  char soundex[5];
-  generateSoundex("AX", soundex);
- // ASSERT_EQ(soundex,"A200");
+TEST(SoundexTestsuite, ReplacesConsonantsWithAppropriateDigits) {
+    char soundex[5];
+    generateSoundex("AX", soundex);
+    ASSERT_STREQ(soundex, "A200");
+}
+
+TEST(SoundexTestsuite, RetainsSoleLetterOfOneLetterWord) {
+    char soundex[5];
+    generateSoundex("A", soundex);
+    ASSERT_STREQ(soundex, "A000");
+}
+
```

Comment:
Good Spread of Test Cases ! Explore Gtest support for DataDriven testing to reduce test efforts